.github/workflows/ci.yml

```diff
@@ -109,8 +109,9 @@ jobs:
     strategy:
       matrix:
         features:
+          - "westend,devnet"
           - "westend,testnet"
-          - "paseo,testnet"
```

Comment:
paseo fails to build on 2412

Reply:
On this front we are updating Paseo codebase as the fellowship updates Polkadot.
And the update to 2412 for Polkadot or Kusama hasn't happened yet.

So unfortunately we may just want to stick with westend on these integration tests for the moment.